services/go-filler/log-fetcher/internal/clients/eth_client.go

```diff
@@ -0,0 +1,19 @@
+package clients
+
+import (
+	"fmt"
+
+	"github.com/base-org/RIP-7755-poc/services/go-filler/log-fetcher/internal/chains"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func GetEthClient(cfg *chains.ChainConfig) (*ethclient.Client, error) {
+	client, err := ethclient.Dial(cfg.RpcUrl)
+	if err != nil {
+		return nil, err
+	}
+
+	fmt.Println("Connected to client")
```

Comment:
Would suggest replacing all of the `fmt.Println` calls with some logger implementation. Projects that rely on geth generally inherit it's logger at `github.com/ethereum/go-ethereum/log`